ecosystem/manager.py

```diff
@@ -346,6 +347,7 @@ def _run_python_tests(
             repo_url: repository url
             tier: tier of project
             python_version: ex: py36, py37 etc
+            package_to_check: qiskit or qiskit-terra
```

Comment:
I'm having a hard time following the code -- where does `package_to_check` end up getting used? I only see it wired through a lot of functions. I'm wondering if we could instead hardcode `qiskit` and remove the argument?

Reply:
It for the beta version.

Without it we can't get the beta version of qiskit (qiskit-terra).
The need is here : https://github.com/mickahell/ecosystem/blob/664d9ccc9ca772ba139ded884cbf466d24808379/ecosystem/templates/configured_tox.ini#L27